pkg/identityserver/mock/application_registry.go

```diff
@@ -87,7 +67,15 @@ func (is *mockIS) ListRights(ctx context.Context, ids *ttnpb.ApplicationIdentifi
 	for _, auth := range auths {
 		if auth == authorization[0] {
 			res.Rights = append(res.Rights,
+				ttnpb.Right_RIGHT_APPLICATION_INFO,
```

Comment:
This is not really equivalent to the old IS mocks. Perhaps `Add()`, should have also the rights attached to it, because right now any key we add has uber-rights.

Reply:
My bad for the delay response, I agree the way it was before indeed gave more rights than needed. 
I took your suggestion and let the rights be passed in the `Add()` method, that way it is easier to make it more granular.

Let me know what you think of the current change